test/lib/ufe/testSelectionByArray.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2020 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import fixturesUtils
+import mayaUtils
+import usdUtils
+
+from maya import cmds
+from maya import standalone
+from maya import mel
+
+import ufe
+
+import unittest
+
+
+def melSelectCmd(paths):
+    """
+    Create the mel command to select the given objects by passing them as a single array.
+    """
+    cmd = '''string $obj[];'''
+    for i, path in enumerate(paths):
+        cmd = cmd + '$obj[%s] = "%s";' % (i, path)
+    cmd = cmd + 'select $obj;'
+    return cmd
+
+
+def melSelect(paths):
+    """
+    Call the mel select command using a single array as argument.
+    """
+    mel.eval(melSelectCmd(paths))
+
+    
+class SelectByArrayTestCase(unittest.TestCase):
```

Comment:
You can actually skip whole classes --- would be appropriate here.  See
https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures
Either that, or make the test conditional in the CMakeLists.txt file.

Reply:
Actually, the test for pure Maya objects can run in older versions, so I'll allow it.